openmmtools/tests/test_integrators.py

```diff
@@ -774,7 +774,7 @@ def run_alchemical_langevin_integrator(nsteps=0, splitting="O { V R H R V } O"):
     if nsigma > NSIGMA_MAX:
         raise Exception("The free energy difference for the nonequilibrium switching for splitting '%s' and %d steps is not zero within statistical error." % (splitting, nsteps))
 
-def test_periodic_langevin_integrator(splitting="V R H O R V", ncycles=40, nsteps_neq=1000, nsteps_eq=1000):
+def test_periodic_langevin_integrator(splitting="H V R O R V H", ncycles=40, nsteps_neq=1000, nsteps_eq=1000):
```

Comment:
outside of the test, would it be advisable to just put an 'H' step at the beginning (or end) of the protocol?  in the AIS regime, i think 'H' would go at the beginning.

Reply:
Just having `H` at the beginning would make the protocol asymmetric, leading to large errors if you are trying to use BAR with one-step protocols. Placing at the beginning and end makes the protocol symmetric, and the bookkeeping automatically splits the perturbation into two smaller pieces. The main difference will be the total number of energy evaluations per integrator cycle.